moveit_ros/planning/planning_request_adapter_plugins/src/add_time_optimal_parameterization.cpp

```diff
@@ -47,12 +47,48 @@ using namespace trajectory_processing;
 class AddTimeOptimalParameterization : public planning_request_adapter::PlanningRequestAdapter
 {
 public:
+  
+  static const std::string RESAMPLE_PARAM_NAME;
+  static const std::string PATH_TOL_PARAM_NAME;
+  static const std::string MIN_ANGLE_PARAM_NAME;
+
   AddTimeOptimalParameterization() : planning_request_adapter::PlanningRequestAdapter()
   {
   }
 
-  void initialize(const ros::NodeHandle& /*nh*/) override
+  void initialize(const ros::NodeHandle& nh) override
   {
+
+    if (!nh.getParam(PATH_TOL_PARAM_NAME, path_tolerance))
```

Comment:
Also this is minor as well, but if you switch the `if(! )` and the `else` it will be a tiny by simpler to read